docs/chat-configuration/ui-customization/incoming-message/carousel.md

```diff
@@ -17,32 +17,34 @@ nav_exclude: true
 
 ---
 
-## Overview
-Displays a "carousel" typed incoming response. 
-As a regular incoming message, carousel has a message section with a timestamp and avatar, and the quick options and channels at the bottom. The carousel provides the ability to display multiple related answers to a user query in a horizontal scrollable structure.
+## Overview 
+Displays the response as a carousel of items. 
+The carousel, just like a regular incoming message, has a message section with a timestamp and avatar, and the quick options and channels at the bottom. The carousel provides the ability to display multiple related answers to a user query in a horizontal scrollable structure.
 {: .overview}
 ![]({{'/assets/images/carousel.png' | relative_url}})
 {: .image-40}
 
 
 ## Carousel Item
-Each item in the carousel consists of 3 sections: 
-- **Image** - If not configured, a placeholder default image will be displayed.
-    Can be configured with a url link. <sub>Optional</sub>.
-- **Info** - Title <sub>Mandatory</sub> and a subtitle <sub>Optional</sub> 
-- **Options** - Item options <sub>Optional</sub>.
+Each corousel item consists of three display properties: 
+- **Image** <sub>(optional)</sub> - THe image can be configured with a URL link. If not image is set, a default image will be displayed.     
+- **Info** <sub>(optional)</sub> - Title <sub>(mandatory)</sub> and a subtitle for the item displayed.
+COMMENT: HOW CAN THIS BE OPTIONAL IS THE TITL IS MANDATORY? 
+- **Options** <sub>(optional)</sub> - Item options .
 
 ---
 
-## UI configurations
-Carousel UI can be configured, by overriding `CarouselItemsUIProvider.configure` method and/or `CarouselItemsUIProvider.customize` method for [dynamic data related customizations]({{'/docs/chat-configuration/ui-cutomization/how-it-works#customize' | relative_url}}).   
-Available configuration proprties can be viewed on `CarouselItemsUIAdapter`
+## Carousel UI configurations
+The carousel UI can be customized, by overriding the `CarouselItemsUIProvider.configure` method and/or the `CarouselItemsUIProvider.customize` method for [dynamic data related customizations]({{'/docs/chat-configuration/ui-cutomization/how-it-works#customize' | relative_url}}).   
+The available configuration proprties can be fund in the `CarouselItemsUIAdapter`.
 
 ### General item configurations
-- Use `CarouselItemsUIAdapter.setCardStyle` method to configure the carousel item look. 
-  > The carousel items display can be configured to have a card like look which has elevation or be flat, with or without rounded corners. 
+- Use the `CarouselItemsUIAdapter.setCardStyle` method to configure the carousel item display. 
+  > The carousel items display configurations are:
+  -  Border-  card like or flat
```

Comment:
this is not the definition of the mentioned method.
The method receives as parameters: radius - for the corners and elevation

Reply:
OK